book/04-git-server/sections/git-daemon.asc

```diff
@@ -41,7 +41,7 @@ respawn
 ----
 
 For security reasons, it is strongly encouraged to have this daemon run as a user with read-only permissions to the repositories – you can easily do this by creating a new user 'git-ro' and running the daemon as them.
-For the sake of simplicity we'll simply run it as the same 'git' user that Gitosis is running as.
+For the sake of simplicity we'll simply run it as the same 'git' user that `git-shell` is using as.
```

Comment:
I know this is closed. I just stumbled upon this while brwosing. Is 'using as' the right phrase? 'running as' seemed better. It doesn't seem grammatical now.